docs/summary.md

```diff
@@ -26,8 +20,16 @@ execute(link, operation).subscribe({
   error: error => console.log(`received error ${error}`),
   complete: () => console.log(`complete`),
 })
+
+//For single execution operations, a Promise can be used
+new Promise((resolve, reject) => (execute(link, operation)).subscribe(resolve, reject))
```

Comment:
I just think most people are going to want a promise in the standalone use case. But we can add it later!